inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -538,6 +538,196 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         return res;
     });
 
+    addSpecificCreator({"PRelu"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "PReLU",
+                             details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<PReLULayer>(attrs);
+        res->params = params;
+
+        const auto weightsNode = node->input_value(1).get_node_shared_ptr();
+        InferenceEngine::details::addBlob(weightsNode, res, InferenceEngine::details::weights);
+
+        return res;
+    });
+
+    addSpecificCreator({"TileIE"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "Tile",
+            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<TileLayer>(attrs);
+        res->params = params;
+        return res;
+    });
+
+    addSpecificCreator({"PriorBoxIE"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "PriorBox",
+            details::convertPrecision(node->get_output_element_type(0))};
+
+        auto res = std::make_shared<CNNLayer>(attrs);
+        res->params = params;
+        res->params["clip"] = res->getBoolStrParamAsIntStr("clip");
+        res->params["flip"] = res->getBoolStrParamAsIntStr("flip");
+        res->params["scale_all_sizes"] = res->getBoolStrParamAsIntStr("scale_all_sizes");
+
+        auto scale_all_sizes = std::stoi(res->params["scale_all_sizes"]);
+        if (!scale_all_sizes) {
+            auto data_pshape = node->get_input_partial_shape(0);
+            if (data_pshape.is_dynamic()) THROW_IE_EXCEPTION << "Dynamic 0-port input of PriorBox is not supported";
+            auto data_shape = data_pshape.to_shape();
+            if (data_shape.size() != 4) THROW_IE_EXCEPTION << "PriorBox has " << data_shape.size() << " items in 0-port input, 4 expected";
+            auto img_pshape = node->get_input_partial_shape(1);
+            if (img_pshape.is_dynamic()) THROW_IE_EXCEPTION << "Dynamic 1-port input of PriorBox is not supported";
+            auto img_shape = img_pshape.to_shape();
+            if (img_shape.size() != 4) THROW_IE_EXCEPTION << "PriorBox has " << data_shape.size() << " items in 1-port input, 4 expected";
+
+            // mxnet-like PriorBox
+            auto img_H = img_shape[2];
+            auto data_H = data_shape[2];
+
+            auto step = std::stof(res->params["step"]);
+            if (step == -1)
+                step = static_cast<float>(1. * img_H / data_H);
```

Comment:
Can we simplify it to:
```suggestion
                step = img_H / static_cast<float>(data_H);
```